x-pack/plugins/security_solution/public/detections/pages/detection_engine/detection_engine.tsx

```diff
@@ -324,33 +307,24 @@ const DetectionEnginePageComponent: React.FC<DetectionEngineComponentProps> = ({
                   {i18n.BUTTON_MANAGE_RULES}
                 </SecuritySolutionLinkButton>
               </HeaderPage>
-              <EuiHorizontalRule margin="m" />
-              <EuiFlexGroup alignItems="center" justifyContent="spaceBetween">
-                <EuiFlexItem grow={false}>
-                  <AlertsTableFilterGroup
-                    status={filterGroup}
-                    onFilterGroupChanged={onFilterGroupChangedCallback}
-                  />
-                </EuiFlexItem>
-
-                <EuiFlexItem grow={false}>
-                  <EuiFlexGroup alignItems="center" gutterSize="none">
-                    <EuiFlexItem grow={false}>
-                      {updatedAt &&
-                        timelinesUi.getLastUpdated({
-                          updatedAt: updatedAt || Date.now(),
-                          showUpdating,
-                        })}
-                    </EuiFlexItem>
-                  </EuiFlexGroup>
-                </EuiFlexItem>
-              </EuiFlexGroup>
-              <EuiSpacer size="m" />
+              <DetectionPageFilterSet
+                dataViewId={dataViewId as string}
```

Comment:
Think it's worth having a placeholder and removing the cast? `dataViewId ? <DetectionPageFilterset ... /> : 'A dataview must be selected to use page filters'`

Reply:
@michaelolo24 , done. No casting now.. The Filter group plugin takes care of invalid dataviewId and displaying the error message.

Reply:
👍🏾 ❤️